#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_extra_qualification.dpatch by  <mozo@sfc.keio.ac.jp>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Avoid extra qualification in the class definition of 
## DP: SBMLSchemaInputSource.h

@DPATCH@

diff -ur libsbml-2.3.4.orig/src/sbml/SBMLSchemaInputSource.h libsbml-2.3.4/src/sbml/SBMLSchemaInputSource.h
--- libsbml-2.3.4.orig/src/sbml/SBMLSchemaInputSource.h	2007-01-19 19:05:12.000000000 +0900
+++ libsbml-2.3.4/src/sbml/SBMLSchemaInputSource.h	2007-01-19 19:05:48.000000000 +0900
@@ -76,7 +76,7 @@
 
   ~SBMLSchemaInputSource ();
 
-  BinInputStream* SBMLSchemaInputSource::makeStream () const;
+  BinInputStream* makeStream () const;
 
 
 private:
